### PR TITLE
Adds CapPa model (https://arxiv.org/abs/2306.07915).

### PR DESCRIPTION
**Pull request by andsteing**
The code was tested on GCP following the instructions in the main [`README`] with the following command

```bash
gcloud compute tpus tpu-vm ssh $NAME --zone=$ZONE --worker=all --command "
cd big_vision
TFDS_DATA_DIR=gs://$GS_BUCKET_NAME/datasets \
bash big_vision/run_tpu.sh big_vision.trainers.proj.cappa.generative \
--config big_vision/configs/proj/cappa/pretrain.py:batch_size=1024 \
--workdir gs://$GS_BUCKET_NAME/big_vision/workdir/`date '+%m-%d_%H%M'`
"
```

[`README`]: https://github.com/google-research/big_vision/blob/main/README.md

### CODE REVIEW

File: big_vision/utils.py [hunk 1]
```diff
@@ -1283,7 +1286,7 @@ def _broadcast(leaf, subtree):
 
 
 def reshard(tree, shardings):
-  """Take an arbitrarily* sharded pytree and shard it according to `shardings`.
+  """Take an *arbitrarily* sharded pytree and shard it according to `shardings`.
```
Comment (lucasb-eyer) [inline big_vision/utils.py:1289]: Can you remove the wrong *?

**Review by lucasb-eyer — APPROVED** [review]
Reviewed proj/-specific code only for leaks or big mistakes, but saw none.

OK with the core code changes that got pulled in.

What seems missing is an update to the global README. It should add a link to the README in the config folder (see other papers), and ideally use the commit hash in the link. I guess you know this and will do it as a follow-up once the hash is available, but you didn't mention it in the PR, so just making sure.

File: big_vision/utils.py (hunk 1 above)
Comment (andsteing) [reply big_vision/utils.py]: Removed.

**Comment by andsteing** [comment]
Confirmed that I will follow up with an update of the global README. Will mention this next time in the PR description.

**Comment by andsteing** [comment]
Updating README in #83

**Comment by sayakpaul** [comment]
Will there be a checkpoint available? 😢

**Pull request merged**